<launch>
    <arg name="distance_tolerance" default="1.0" />
    <arg name="file_name" default="waypoints" />
    <node name="publish_marker" pkg="waypoints_generator_csv" type="publish_marker.py" output="screen">
        <param name="file_name" value="$(arg file_name)" />
        <param name="distance_tolerance" value="$(arg distance_tolerance)" />
        <param name="path_to_pkg" value="$(find waypoints_generator_csv)" />
    </node>
</launch>